application/views/scripts/index/index.phtml

```diff
@@ -13,5 +13,5 @@
       <td><a target="graph_iframe" class="icon-clock" href="<?= urldecode($this->url) ?>&from=-1y">1 year</a></td>
     </tr>
   </table>
-  <iframe name ="graph_iframe" src="<?= urldecode($this->url) ?>&from=-1d" style="height: 700px; width: 800px" frameborder="no" scrolling="no" marginwidth="0" marginheight="0" hspace="0" vspace="0"></iframe>
+  <iframe name ="graph_iframe" src="<?= urldecode($this->url) ?>&from=-1d" style="height: 800px; width: 1024px" frameborder="no" scrolling="no" marginwidth="0" marginheight="0" hspace="0" vspace="0"></iframe>
```

Comment:
Not sure the iframe should be re sized, it might break people. Perhaps it should be configurable?